eclair-core/src/main/scala/fr/acinq/eclair/channel/ChannelTypes.scala

```diff
@@ -476,61 +485,11 @@ final case class RemoteParams(nodeId: PublicKey,
                               paymentBasepoint: PublicKey,
                               delayedPaymentBasepoint: PublicKey,
                               htlcBasepoint: PublicKey,
-                              features: Features)
+                              features: Features,
+                              shutdownScript: Option[ByteVector])
```

Comment:
The naming is inconsistent `LocalParams.defaultFinalScriptPubKey` and `RemoteParams.shutdownScript`

Either:
- `LocalParams.defaultFinalScriptPubKey` and `RemoteParams.upfrontFinalScriptPubKey`
or
- `LocalParams.defaultShutdownScript` and `RemoteParams.upfrontShutdownScript`

I would keep the `upfront` in any case.

Reply:
Let's revisit the naming afterwards.